### PR TITLE
fix(DB/Creature): Fix static Ironjaw Basilisks

### LINKED ISSUE #6899: Missing Movement - Several Ironjaw Basilisks, Stranglethorn Vale
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [ ] enUS
- [X] other frFR

##### FACTION

- [ ] Alliance
- [X] Horde

##### CONTENT PHASE:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [X] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

Original report: https://github.com/chromiecraft/chromiecraft/issues/1166

Some Ironjaw Basilisk  is standing still without any movement. <provided GUIDS omitted as they were incorrect.>

https://user-images.githubusercontent.com/82708448/125193911-39814900-e24f-11eb-9a94-94b2dc781fe9.mp4

@jamadaha adds: 
The specified GUID does not match any Ironjaw Basilisk on my local instance. (AzerothCore 9b3993b) I did, however, find the Basilisk shown in the video (GUID 756) and it does indeed not move in the five minutes that I looked at it. (Very interesting stuff I must say)
image

Tester: Checked the DB for static Ironjaws, found 5:
```
+------+
| guid |
+------+
|  373 |
|  612 |
|  756 |
|  981 |
| 1062 |
+------+
5 rows in set (0.00 sec)
```

##### EXPECTED BLIZZLIKE BEHAVIOUR:

The Ironjaw Basilisk should roam a bit and not be static.

##### SOURCE:

https://www.youtube.com/watch?v=sVhBwN0EsB0 - footage of someone farming them here - didn't spot any static ones, though it's not ideal for checking.

##### STEPS TO REPRODUCE THE PROBLEM:

1. Port to GUIDs 373, 612, 756, 981, or 1062.

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/7c21d567d59153b37978756a5bf02daf6998e665

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

### PR DESCRIPTION
**Pull request by jamadaha**
- The Ironjaw Basilisks (GUID 373, 612, 756, 981 and 1062) were the only non moving in the cave, whereas they now wander.
 - Closes #6899

## Changes Proposed:
-  Add wandering to the Ironjaw Basilisks (GUID 373, 612, 756, 981 and 1062).

## Issues Addressed:

- Closes #6899

## Tests Performed:

- .go c 373/612/756/981/1062
- Observe

## How to Test the Changes:

1. .go c 373/612/756/981/1062
2. Observe
3.

## Known Issues and TODO List:

N/A

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1626100751770962700.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1626100751770962700');
+
+-- Add wandering to the NPCs 373, 612, 756, 921 and 1062
+UPDATE `creature` SET `MovementType` = 1, `wander_distance` = 2 WHERE `id` = 1551 AND `guid` = 373;
+UPDATE `creature` SET `MovementType` = 1, `wander_distance` = 2 WHERE `id` = 1551 AND `guid` = 612;
+UPDATE `creature` SET `MovementType` = 1, `wander_distance` = 5 WHERE `id` = 1551 AND `guid` = 756;
+UPDATE `creature` SET `MovementType` = 1, `wander_distance` = 5 WHERE `id` = 1551 AND `guid` = 981;
+UPDATE `creature` SET `MovementType` = 1, `wander_distance` = 5 WHERE `id` = 1551 AND `guid` = 1062;
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1626100751770962700.sql:8]: Please add new line at the end of file

**Review by LukasVolgger — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1626100751770962700.sql (hunk 1 above)
Comment (jamadaha) [reply data/sql/updates/pending_db_world/rev_1626100751770962700.sql]: Should be fixed now

**Review by LukasVolgger — APPROVED** [review]
Either my app or Github has a problem, sry it was fine!

**Comment by Eldarix** [comment]
Checked (AC HASH/COMMIT: 05bbf4a), they move correctly I would say ;)

WoWScrnShot_071321_111454
WoWScrnShot_071321_111513
WoWScrnShot_071321_111537
WoWScrnShot_071321_111554
WoWScrnShot_071321_111618

**Comment by LukasVolgger** [comment]
Thx @jamadaha for your contribution and @Eldarix for testing!

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**